app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,7 @@
         <% end %>
       </div>
       <div class="mu-navbar-avatar">
-        <% if current_user? %>
+        <% if current_user? && !current_user.incognito? %>
```

Comment:
It's not necessary now but maybe we should refactor a bit this section and move it to a helper to make it `incognitable` :stuck_out_tongue: as well without adding this condition